orchestrator-salt/src/main/resources/salt-common/salt/monitoring/template/cdp-request-signer.yaml.j2

```diff
@@ -1,5 +1,13 @@
 {%- from 'telemetry/settings.sls' import telemetry with context %}
 {%- from 'monitoring/settings.sls' import monitoring with context %}
+{%- set instance_name = salt['grains.get']('fqdn') %}
+{%- if telemetry.platform in ["AWS", "AZURE", "GCP"]  %}
+  {%- set get_instance_cmd = "python3 /opt/salt/scripts/instanceid_retriever.py " +  telemetry.platform %}
+  {%- set instance_cmd_output = salt.cmd.run(get_instance_cmd) %}
```

Comment:
Sanity check: Is it usual practice to run scripts like this, instead of having a dedicated state for the execution?

Reply:
just if you need to gather/calculate the data from the node and not passing through as models from server (like to put in pillar), note this data does not exist in freeipa node, but still it's more reliable from the metadata service